src/script/lua_api/l_env.cpp

```diff
@@ -359,6 +359,39 @@ int ModApiEnv::l_get_node_or_nil(lua_State *L)
 	return 1;
 }
 
+// bulk_get_node([pos1, pos2, ...])
+// pos = {x=num, y=num, z=num}
+int ModApiEnv::l_bulk_get_node(lua_State *L)
+{
+	GET_ENV_PTR;
+
+	// parameters
+	if (!lua_istable(L, 1)) {
+		return 0;
```

Comment:
should probably be an error?

Reply:
There is no error in this case in `l_bulk_set_node`.

So let's add an error message for `l_bulk_set_node` and for `l_bulk_get_node` or keep it without it both.
What is a preferred solution?

Reply:
Guess not then.